Fix review area not collapsing in popup

cha5on opened the pull request:
Reviews section was not collapsing because it wasn't testing
vm.itemDetails.  Rather than testing that in every div, place all of the
detail divs inside of a new div that toggles with vm.itemDetails.
No other changes to that block besides indentation for new div.

Also, remove invocation of nonexistent function vm.changeDetails().

Fixes #2367.

cha5on commented:
Here's what it looks like collapsed on both the Overview and Review tab now.  Also note the hidden Overview/Review selector.

![image](https://user-images.githubusercontent.com/1630923/33236838-000453f0-d215-11e7-8b4c-e98b208c714a.png)

duckmanBR commented on src/app/move-popup/dimMoveItemProperties.html:
```diff
@@ -39,35 +39,37 @@
-    <div class="item-details item-sockets" ng-if="vm.item.sockets && vm.itemDetails">
-      <sockets sockets="vm.item.sockets" infuse="vm.infuse(vm.item, $event)"></sockets>
+  <div ng-if="vm.itemDetails" class="move-popup-details">
+    <div ng-if="vm.item.reviewable" class="move-popup-tabs">
```
While there, also include a check to (I believe it's) $ctrl.canReview to only show the selectors if the "Show detailed weapon reviews and allow rating weapons" is checked and fix #2368

cha5on replied:
I actually think the discoverability of detailed reviews was a good idea, so I'm keeping the options there and fixing the display issue.

cha5on commented:
Rebased on top of tree, and fixed #2368 as suggested by @duckmanBR (see screenshot below)

![image](https://user-images.githubusercontent.com/1630923/33260242-12527f82-d314-11e7-91e6-f8739e1dd7b9.png)

bhollis commented:
Thanks!
